docs/install/Fedora.md

```diff
@@ -14,7 +14,7 @@ dnf install atomic
 ##Make
 On Fedora, You need to install required build dependencies
 ```
-yum-builddep atomic
+#yum-builddep atomic
```

Comment:
Should probably change this to 

dnf builddep atomic

Since yum is going away on Fedora